tox.ini

```diff
@@ -1,5 +1,5 @@
 [tox]
-envlist = py{36,37,38,39}-dj{22,31,32}
+envlist = py{36,37,38,39,310}-dj{22,31,32}
```

Comment:
idk how tox works but 3.10 is only supported in 3.2 is that an issue in this config?

Reply:
![image](https://user-images.githubusercontent.com/5583319/137713977-291262f8-ddef-4c35-bca1-c6dcc362bc0d.png)